integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/environment/EnvironmentStopStartTests.java

```diff
@@ -50,24 +62,30 @@ public void testCreateStopStartEnvironment(TestContext testContext) {
                 .given(CredentialTestDto.class)
                 .when(credentialTestClient.create())
                 .given("telemetry", TelemetryTestDto.class)
-                .withLogging()
-                .withReportClusterLogs()
+                    .withLogging()
+                    .withReportClusterLogs()
                 .given(EnvironmentTestDto.class)
-                .withNetwork()
-                .withTelemetry("telemetry")
-                .withCreateFreeIpa(Boolean.TRUE)
+                    .withNetwork()
+                    .withTelemetry("telemetry")
+                    .withCreateFreeIpa(Boolean.TRUE)
+                    .addTags(ENV_TAGS)
                 .when(environmentTestClient.create())
                 .await(EnvironmentStatus.AVAILABLE)
+                .then(cloudProviderSideTagAssertion.verifyEnvironmentTags(ENV_TAGS))
                 .given(SdxInternalTestDto.class)
+                    .addTags(SDX_TAGS)
                 .when(sdxTestClient.createInternal())
                 .awaitForFlow(RunningParameter.key(resourcePropertyProvider().getName()))
                 .await(SdxClusterStatusResponse.RUNNING)
+                .then(cloudProviderSideTagAssertion.verifyInternalSdxTags(SDX_TAGS))
```

Comment:
Can we also verify here the Tags that are getting added by Default fro the Account Level tag configuration?
Like the default Owner tag.